packages/kotti-ui/source/kotti-style/forms/_toggles.scss

```diff
@@ -0,0 +1,6 @@
+.form-checkbox .form-icon {
```

Comment:
Shouldn't this be scoped to only apply to the table?

Reply:
hum it can. It's currently necessary only for KtTable if I'm not wrong.
Here I just put it back how it was originally